server/notification-providers/grafana-oncall.js

```diff
@@ -0,0 +1,54 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { DOWN, UP } = require("../../src/util");
+
+class GrafanaOncall extends NotificationProvider {
+
+    name = "GrafanaOncall";
+
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
+        try {
+
+            // If heartbeatJSON is null, assume we're testing.
+            if (heartbeatJSON == null) {
+                let GrafanaOncallTestData = {
+                    title: "This is a test",
+                    message: msg,
+                    state: "alerting",
+                };
+                await axios.post(notification.grafanaoncallURL, GrafanaOncallTestData);
+                return okMsg;
+            }
+
+            if (heartbeatJSON["status"] === DOWN) {
+                let grafanadowndata = {
+                    title: monitorJSON["name"] + " is down",
+                    message: heartbeatJSON["msg"],
+                };
```

Comment:
Do we not need to set `state: "alerting"` here? [docs](https://grafana.com/docs/oncall/latest/integrations/available-integrations/configure-webhook/)

```suggestion
                let grafanadowndata = {
                    title: monitorJSON["name"] + " is down",
                    message: heartbeatJSON["msg"],
                    state: "alerting",
                };
```

Reply:
Ok, since it's not needed by Oncall i will remove it.